content/300-guides/125-development-environment/050-environment-variables/index.mdx

```diff
@@ -77,14 +77,30 @@ Refer to the `dotenv` documentation for information about [what happens if an en
 
 ### Expanding variables
 
-Variables stored in `.env` files can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+Variables stored in `.env` files can be expanded using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). 
 
 ```
 # .env
 DATABASE_URL=postgresql://test:test@localhost:5432/test
 DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
 ```
```

Comment:
````suggestion
```env file=.env
DATABASE_URL=postgresql://test:test@localhost:5432/test
DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
```
````

Reply:
Do we have an example in the docs already what that looks like? Would like to see it this time before merging if possible :D

Reply:
Sure:
![fileenv](https://user-images.githubusercontent.com/22930449/131672015-e9bb904c-66b1-4912-b7f2-c1df2b0012e1.PNG)